launchable/commands/record/tests.py

```diff
@@ -208,6 +225,38 @@ def parse_func(self) -> ParseFunc:
         def parse_func(self, f: ParseFunc):
             self._parse_func = f
 
+        @property
+        def build_name(self) -> str:
+            return self._build_name
+
+        @build_name.setter
+        def build_name(self, build_name: str):
+            self._build_name = build_name
+
+        @property
+        def test_session_id(self) -> int:
+            return self._test_session_id
+
+        @test_session_id.setter
+        def test_session_id(self, test_session_id: int):
+            self._test_session_id = test_session_id
+
+        @property
+        def session_id(self) -> str:
+            return self._session_id
+
+        @session_id.setter
+        def session_id(self, session_id: str):
+            self._session_id = session_id
```

Comment:
[question]
Is the test_session_id different from the session_id?

Reply:
Yes 

In this codes, we call this format `builds/<build number>/test_sessions/123` as `session_id`. And `123` as `test_session_id`